orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/StageDefinitionBuilder.java

```diff
@@ -50,6 +50,10 @@ default void taskGraph(
     return emptyList();
   }
 
+  default @Nonnull List<Stage> afterStages(@Nonnull Stage stage) {
```

Comment:
My expectation is that we can subsequently start moving `STAGE_AFTER` definitions from `aroundStages()` to `afterStages()`.

Reply:
What happens if you have after stages in both methods?

Reply:
Your `afterStages()` would not get planned ... the `CompleteStageHandler` first calls `Stage.firstAfterStages()` to verify that there aren't any synthetic after stages to begin with.